.github/workflows/pytorch.yml

```diff
@@ -63,3 +63,9 @@ jobs:
         python pytorch/pytorch_lightning_simple.py
       env:
         OMP_NUM_THREADS: 1
+    - name: Run PyTorch Lightning DDP examples
+      run: |
+        python pytorch/pytorch_lightning_ddp.py
+        python pytorch/pytorch_lightning_ddp.py -p
```

Comment:
The pruning version seems enough to be tested with the CI. In addition, it might be faster than without pruning. However, I think other callback integrations are tested without the pruning option. So, I'd like to hear the other dev members opinion...

Reply:
I agree that it's open for discussion. The total code coverage, including the code path in Optuna, would be improved with pruning turned on. The code coverage inside the example would basically be unchanged thought, and the integration modules are tested in Optuna's main repository. This might be the reasons for the current test configurations in this repository. How about discussing that in a different issue?

Reply:
Thank you for your comment! Sounds reasonable. I'll merge this PR as is the current version and I'll create an issue to discuss it.